onnxruntime/core/providers/vitisai/imp/tensor_proto.h

```diff
@@ -1,10 +1,25 @@
 // Copyright (c) 2023 Advanced Micro Devices, Inc. All rights reserved.
 // Licensed under the MIT License.
 #pragma once
-//
+
 #include "core/common/gsl.h"
 #include "onnx/onnx_pb.h"
+#include "core/providers/dml/DmlExecutionProvider/src/External/D3DX12/d3dx12.h"
+#include "core/providers/dml/DmlExecutionProvider/src/ErrorHandling.h"
+
+#include <cstdint>
+#include <limits>
+#include <wrl.h>
+#include <wil/Resource.h>
+#include <d3d12.h>
+
+#define FENCE_SIGNAL_VALUE 1
+
+
 namespace vaip {
+using Microsoft::WRL::ComPtr;
```

Comment:
Same here. I prefer to keep this from the global namespace. This header file can be included from any place in the source code.